src/app/users/users-routing.module.ts

```diff
@@ -41,15 +42,24 @@ const routes: Routes = [
           }
         },
         {
-          path: ':id',
-          data: { title: extract('View User'), routeResolveBreadcrumb: ['user', 'username'] },
-          resolve: {
-            user: UserResolver
-          },
+          path: 'view/:id',
```

Comment:
Can you elaborate why have you changed the routing? Try to make it consistent with other components of the web app.

Reply:
The `/view/id` routing path is closer (visually) to the one on the community app (e.g. https://demo.mifos.io/#/viewuser/2152). This format is also used in other parts of the app like for example in the `accounting-routing.module.js` or `self-service-routing.module.js`.

Reply:
Yeah, I saw that then wait for the maintainers to review it :)

Reply:
restore routing to previous state, we don't just have to make it similar we have to evolve to better ways as well.